### PR TITLE
changed date predicate to be today's date

### PR DESCRIPTION
**Pull request by wadeking98**
Resolves: #17
Signed-off-by: wadeking98 <wkingnumber2@gmail.com>

### CODE REVIEW

File: server/src/content/student/useCases/OnlineStore.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { StepType } from '../../types'
 const URL = '/public/student/useCases/store'
 
 const date = new Date()
-date.setFullYear(date.getFullYear() - 18)
+date.setFullYear(date.getFullYear())
```
Comment (cvarjao) [inline server/src/content/student/useCases/OnlineStore.ts:10]: can this line just be removed if setting fullYear to itself?

**Review by amanji — COMMENTED** [review]
Initially this was supposed to check whether the expiry date on the student ID has not passed four years from todays date. It's seems like it's checking against age.

File: server/src/content/student/useCases/OnlineStore.ts (hunk 1 above)
Comment (wadeking98) [reply server/src/content/student/useCases/OnlineStore.ts]: you're right, I just updated it

**Comment by amanji** [comment]
Yes this line can just be removed 👍

**Review by amanji — APPROVED** [review]

**Pull request merged**